be/src/vec/exec/volap_scan_node.cpp

```diff
@@ -34,6 +34,32 @@ VOlapScanNode::VOlapScanNode(ObjectPool* pool, const TPlanNode& tnode, const Des
     _free_blocks.reserve(_max_materialized_blocks);
 }
 
+Status VOlapScanNode::init(const TPlanNode& tnode, RuntimeState* state) {
+    assert(state);
```

Comment:
Do not need to check here. If state is not set, then many method will crashed. We do not need to check every condition.